game/src/ecs/systems/VelocitySystem.java

```diff
@@ -55,6 +57,19 @@ private VSData buildDataObject(VelocityComponent vc) {
     }
 
     private void movementAnimation(Entity entity) {
+
+        AtomicBoolean isDead = new AtomicBoolean(false);
```

Comment:
Warum `AtomicBoolean`?
Müssen wir hier Thread-Safe sein?

Reply:
Das hat in diesem Fall nix mit Thread-Safe zutun, sondern ich kann eine normale Variable nicht von innerhalb eines Lamdbda-Ausdruck ändern, selbst lesender zugriff geht (meine ich, ungetestet) nur wenn die Variable final ist.